### ISSUE TITLE
[DESIGN] Increase contrast on axis labels

### ISSUE THREAD

**Issue by laurendoone** [Hyperobjekt/seda-map#436]
Feedback from James: Axis labels here are too light and fail contrast accessibility requirements. Don't really see other examples from a visual inspection, but we should be mindful about contrast ratio.
![image](https://user-images.githubusercontent.com/76966798/104643776-bdbc9000-5661-11eb-8f08-e4bb8a24ce65.png)
